src/OmniSharp.Shared/Utilities/PhysicalFileProviderExtensions.PollingFileChangeToken.cs

```diff
@@ -118,10 +118,10 @@ public bool ActiveChangeCallbacks
 
             public IDisposable RegisterChangeCallback(Action<object> callback, object state)
             {
-                if (state != null)
-                {
-                    throw new ArgumentException($"Stateful callbacks are not supported for {nameof(PollingFileChangeToken)}", nameof(state));
-                }
+                // if (state != null)
```

Comment:
I saw this exception in my Cake LSP server when pulling in O#. Can't remember how I fixed it though.